ci/pipelines/deploy-to-production.yml

```diff
@@ -1498,20 +1499,58 @@ jobs:
 
   - name: retag-adminusers-image-for-test-perf-db
     plan:
-      - get: pay-ci
-      - get: adminusers-ecr-registry-prod
-        params:
-          format: oci
-        passed: [adminusers-db-migration-prod]
-        trigger: true
-      - task: parse-perf-release-tag
-        file: pay-ci/ci/tasks/parse-perf-db-release-tag.yml
-        input_mapping:
-          ecr-repo: adminusers-ecr-registry-prod
-      - put: adminusers-ecr-registry-test
+      - in_parallel:
+          steps:
+          - get: adminusers-ecr-registry-test
```

Comment:
This could potentially lead to some confusion. If multiple PRs are merged, there could be a new image in test ECR by the time the first one is deployed in production.

For example `adminusers-ecr-registry-prod` could return `999` and due to a new release `adminusers-ecr-registry-test` could return `1000`

or I am missing something?

Reply:
I agree. I keep changing my mind about the way this task ought to be done. Nothing seems quite right.

Reply:
No, the release number of the image in test ECR comes from the release number of the image in prod ECR.

Reply:
Yeah you shouldn't need the test one, you can be sure that any XX-release in prod has an XX-candidate in test, so it should be enough to parse the XX-release from prod and just implicitly know it's there in test in order to create a new manifest from of XX-perf